### DISCUSSION TITLE
Repair command issues

### DISCUSSION THREAD

**Discussion by GlowheadS** [Zrips/CMI#7048] [General]
Hello there,

I'm using LuckPerms & I'm trying to grant a specific rank the /repair command. However after adding the permission cmi.command.repair it doesn't seem to be working ??

**Comment by mrfloris** [comment]
<img width="848" alt="Screen-Shot 2022-09-25 at 18 16 59" src="https://user-images.githubusercontent.com/28841349/192153931-8562f942-76a1-4441-b1c3-923cff61ebac.png">
<img width="1386" alt="Screen-Shot 2022-09-25 at 18 17 44" src="https://user-images.githubusercontent.com/28841349/192153969-e71a1a9e-48c6-4502-85bb-63ee081ae200.png">

Try again in the latest cmi and cmilib combo, (upgrade if you haven't already), and see if these commands/permissions and configurations work for you (apply the settings to your liking of course, I'm just showing what I can find). and see if it works for you now.
